lib/sidekiq/cli.rb

```diff
@@ -231,6 +231,10 @@ def boot_system
           end
           require 'sidekiq/rails'
           require File.expand_path("#{options[:require]}/config/environment.rb")
+
+          if ::Rails::VERSION::MAJOR > 4 && ::Rails.env.development?
```

Comment:
Thanks for the feedback!

On Wed, Jun 15, 2016 at 4:15 PM, Edward Rudd <notifications@github.com>
wrote:

> In lib/sidekiq/cli.rb
> <https://github.com/mperham/sidekiq/pull/2457#discussion_r67263057>:
>
> > @@ -231,6 +231,10 @@ def boot_system
> >            end
> >            require 'sidekiq/rails'
> >            require File.expand_path("#{options[:require]}/config/environment.rb")
> > +
> > +          if ::Rails::VERSION::MAJOR > 4 && ::Rails.env.development?
>
> Shouldn't this check if Rails.configuration.cache_classes is set to false
> instead of relying on a specific environment name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mperham/sidekiq/pull/2457/files/e4d09527c43816706587d86eef3b4036128b465e#r67263057>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAALXwBfTjj6BiHKfUxGjfJwQiElR9Zeks5qMIedgaJpZM4Ffbrt>
> .
>